features/sriov/10-sriov-networknodepolicy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: ${nic}
```

Comment:
Here we are using ${nic} for several different reasons:
- Node policy name
- Resource name
- Pf Name(s)

Can we have them different, so it's clear they are not the same thing?

Reply:
Fixed.